tmt/steps/execute/internal.py

```diff
@@ -124,11 +142,13 @@ def _test_environment(self, test, extra_environment):
 
         return environment
 
-    def execute(self, test, guest, extra_environment):
+    def execute(self, test: Test, guest: Guest, progress: str,
```

Comment:
Why is the `progress` parameter added here but never used in the `execute` method?

Reply:
No idea. Removed. :)

Reply:
Ok :)